app/index.html

```diff
@@ -46,7 +46,7 @@
     <link rel="stylesheet" href="styles/main.css">
     <!-- endbuild -->
   </head>
-  <body ng-app="faqApp" class="faq blue">
```

Comment:
My bad, I moved them to the ui-view tag to get around what I thought was an issue where Angular would try and take over, then the noscript wouldn't work because it needed Angular to load it, but it turns out that I just had an environment issue. Let me move them back to here.